components/question/ScrollingSelect/index.jsx

```diff
@@ -0,0 +1,139 @@
+  const [activeOption, setActiveOption] = useState(null);
+  const isMultipleSelect = pick === 'any';
+
+  useEffect(() => {
+    setSelectOptions(buildSelectOptions(survey, answers));
+  }, [id, survey, answers]);
+
+  useEffect(() => {
+    setActiveOption(selectOptions[0]?.value);
+  }, [selectOptions]);
+
+  const sliderRef = useRef();
+  const scroll = useCallback(
+    y => {
+      if (y > 0) {
+        return sliderRef?.current?.slickNext();
+      } else {
+        return sliderRef?.current?.slickPrev();
+      }
+    },
+    [sliderRef],
+  );
+
+  useEffect(() => {
+    window.addEventListener('wheel', e => {
+      scroll(e.deltaY);
+    });
+  }, [scroll]);
+
+  const onOptionSelect = index => {
+    const newSelectOptions = selectOptions.slice();
+    newSelectOptions[index].selected = !newSelectOptions[index].selected;
+    setSelectOptions(newSelectOptions);
+
+    const answerId = newSelectOptions[index].value;
+    const surveyQuestionsWithAnswers = buildSurveyQuestionsWithAnswers(id, answerId);
+    const newAnswerObject = surveyQuestionsWithAnswers.answers[0];
+
+    setSurveyQuestionsWithAnswers(prevSurveyQuestionWithAnswer => {
+      let newSurveyQuestionWithAnswer = prevSurveyQuestionWithAnswer.slice();
+      const questionObject = newSurveyQuestionWithAnswer.find(response => response.id === id);
+
+      if (!isMultipleSelect) {
+        newSurveyQuestionWithAnswer = newSurveyQuestionWithAnswer.filter(response => response.id !== id);
+        return [...newSurveyQuestionWithAnswer, surveyQuestionsWithAnswers];
+      }
+
+      if (!questionObject) {
+        return [...newSurveyQuestionWithAnswer, surveyQuestionsWithAnswers];
+      }
+
+      let answerObject = questionObject.answers.find(answer => answer.id === answerId);
+      if (answerObject) {
+        questionObject.answers = questionObject.answers.filter(answer => answer.id !== answerId);
+      } else {
+        questionObject.answers.push(newAnswerObject);
+      }
+
+      return newSurveyQuestionWithAnswer;
+    });
+  };
+
+  const renderCheckBox = option => {
+    if (!isMultipleSelect) {
+      return null;
+    } else if (option.selected) {
+      return (
+        <Image
+          className={classNames({ 'opacity-50': activeOption !== option.value })}
+          src={circleCheckIcon}
+          alt="circle check icon"
+          data-test-id={scrollingSelectTestIds.icon}
+        />
+      );
+    } else {
+      return <Image src={circleCheckEmptyIcon} alt="circle check empty icon" data-test-id={scrollingSelectTestIds.icon} />;
+    }
+  };
+
+  const sliderEvents = { afterChange: currentIndex => setActiveOption(selectOptions[currentIndex].value) };
+  const sliderConfig = { ...sliderSettings, ...sliderEvents };
+
+  return (
+    <div data-test-id={scrollingSelectTestIds.base}>
+      <Slider {...sliderConfig} ref={sliderRef} className={styles.scrollingSelect} data-test-id={scrollingSelectTestIds.base}>
```

Comment:
If there are more than 2 options, it should add the border to the center item, I think.
And for the multiple selections, I feel a bit jerky:
![2022-05-19 12 12 49](https://user-images.githubusercontent.com/7344405/169213914-0a2e6f5c-94db-47a8-b97d-299528431a3f.gif)
Could you check again? I think for the multiple selection, we just highlight the center item only 🤔

Reply:
@hoangmirs , added border to the center item in e278afe889ee3629e9c2704916fcadd269ca9da4!
Regarding the jerk, I think you've tested in laptop trackpad? I described an issue with the laptop trackpad in the `insight` section of this PR's description!